sw/device/silicon_creator/rom_ext/e2e/ownership/BUILD

```diff
@@ -303,7 +303,7 @@ ownership_transfer_test(
 ownership_transfer_test(
```

Comment:
To understand ownership and the ownership transfer tests, see the following:

//sw/device/silicon_creator/rom_ext/doc/ownership.md
//sw/device/silicon_creator/rom_ext/data/rom_ext_e2e_testplan.hjson